Source/SourceKittenFramework/WindowsError.swift

```diff
@@ -0,0 +1,32 @@
+#if os(Windows)
+import WinSDK
+
+@_transparent
+internal func MAKELANGID(_ p: WORD, _ s: WORD) -> DWORD {
+    return DWORD((s << 10) | p)
+}
+
+struct WindowsError {
+    public let code: DWORD
+}
+
+extension WindowsError: Error {
+    var localizedDescription: String {
+        let dwFlags: DWORD = DWORD(FORMAT_MESSAGE_ALLOCATE_BUFFER)
+                           | DWORD(FORMAT_MESSAGE_FROM_SYSTEM)
+                           | DWORD(FORMAT_MESSAGE_IGNORE_INSERTS)
```

Comment:
This should fix one of the SwiftLint violations:

```suggestion
        let dwFlags = DWORD(FORMAT_MESSAGE_ALLOCATE_BUFFER)
                    | DWORD(FORMAT_MESSAGE_FROM_SYSTEM)
                    | DWORD(FORMAT_MESSAGE_IGNORE_INSERTS)
```